models/gorm_repository.go

```diff
@@ -206,14 +204,14 @@ var wellKnown = map[string]*WorkItemType{
 		ID:   1,
 		Name: "system.workitem",
 		Fields: map[string]FieldDefinition{
+			"system.title": FieldDefinition{Type: SimpleType{Kind: KindString}, Required: true},
 			"system.owner": FieldDefinition{Type: SimpleType{Kind: KindUser}, Required: true},
```

Comment:
The `owner` field has been renamed to `assignee` in #185. Remember to change it once this gets merged.